ArduCopter/Log.cpp

```diff
@@ -552,26 +498,6 @@ const struct LogStructure Copter::log_structure[] = {
       "HELI",  "Qffff",        "TimeUS,DRRPM,ERRPM,Gov,Throt", "s----", "F----" },
 #endif
 
-// @LoggerMessage: PL
```

Comment:
Why do you delete this information instead of moving it to another file?

Reply:
> Why do you delete this information instead of moving it to another file?

Oh, I do - but then I forget to add the file, and the rest is history.

Thankfully, now-adays history can be rewritten and I've just done that ;-)